txpool/event_manager_test.go

```diff
@@ -112,6 +112,8 @@ func TestEventManager_SignalEvent(t *testing.T) {
 
 	completed := false
 	for !completed {
+		delay := time.NewTimer(5 * time.Second)
```

Comment:
I would suggest using one timer and `Reset()` here.

Reply:
085bf03 replace it